src/Microsoft.TestPlatform.Common/DataCollection/DataCollectionManager.cs

```diff
@@ -628,5 +658,40 @@ private void LogAttachments(List<AttachmentSet> attachmentSets)
                 }
             }
         }
+
+        /// <summary>
+        /// Initializes the Data Collector Cache.
+        /// </summary>
+        /// <param name="extensionsPath">Extensions path.</param>
+        private void InitializeDataCollectorCache(string extensionsPath)
```

Comment:
This doesn't follow the existing extensibility convention of `TestPluginManager` etc.. Is it intentional?